### DISCUSSION TITLE
Folder Organization Feature for Projects in Studio Page

### DISCUSSION THREAD

**Discussion by hasanals** [langgenius/dify#3619] [Suggestion]
Hello Dify Team,

I would like to request a feature that would be a great addition to the app.

> Is this request related to a challenge you are facing?

Yes, as a user who creates many projects, navigating through them has become a challenge due to the lack of organization options.

> What is the feature you'd like to see?

I propose the addition of a "Create Folder" button in the "Create App" section. This feature would allow users to create folders and move projects into them for better organization.

> How will this feature improve your workflow / experience?

This feature would significantly improve my workflow by providing a more organized view of my projects. It would make navigation easier and more efficient, enhancing my overall experience with Dify.

**Comment by GeorgeLxw** [comment]
In addition, it is hoped that the permissions of each application and folder can be controlled.
